conda/build.sh

```diff
@@ -1,12 +1,20 @@
+#! /usr/bin/env bash
+set -euo pipefail
+
+# Cd into FF_HOME
+cd "${BASH_SOURCE[0]%/*}/../"
+
 # build flexflow
-export CXXFLAGS=$(echo $CXXFLAGS | sed 's/-O2//g')
-export CXXFLAGS=$(echo $CXXFLAGS | sed 's/-std=c++17//g')
-export CXXFLAGS=$(echo $CXXFLAGS | sed 's/-DNDEBUG//g')
-export CXXFLAGS=$(echo $CXXFLAGS | sed 's/-D_FORTIFY_SOURCE=2//g')
-export CPPFLAGS=$(echo $CPPFLAGS | sed 's/-O2//g')
-export CPPFLAGS=$(echo $CPPFLAGS | sed 's/std=c++17//g')
-export CPPFLAGS=$(echo $CXXFLAGS | sed 's/-DNDEBUG//g')
-export CPPFLAGS=$(echo $CXXFLAGS | sed 's/-D_FORTIFY_SOURCE=2//g')
+CXXFLAGS="${CXXFLAGS//-O2/}"
```

Comment:
Might be nice to include a link to https://wiki-dev.bash-hackers.org/syntax/pe here for people not familiar with this syntax? Up to you :slightly_smiling_face: